scripts/ci/update-generator/main.go

```diff
@@ -140,32 +128,33 @@ func createInstaller(outputPath, channel, platform string) error {
 }
 
 func run() error {
+	var (
+		platform = fetchPlatform()
+		inDir    = defaultInputDir
+		outDir   = defaultOutputDir
+		branch   = constants.BranchName
+		version  = constants.Version
+	)
+
+	flag.StringVar(
+		&platform, "platform", platform,
+		"Target platform in the form OS-ARCH. Defaults to running os/arch or the combination of the environment variables GOOS and GOARCH if both are set.",
+	)
+	flag.StringVar(&inDir, "i", inDir, "Override directory to gather payload from.")
```

Comment:
Same here

Reply:
I think we should keep this one available since the output location of the payload-generator script can be set, and this input flag would then necessarily need to be set.